cypress/integration/ambianic-tests/timeline.spec.js

```diff
@@ -3,7 +3,8 @@
 context('Timeline', () => {
     before(() => {
       cy.visit('http://localhost:8080')
-      cy.get('[data-cy=timeline]').click()
+    //   cy.get('[data-cy=timeline]').click()
```

Comment:
@vickywane same comment here. What is the point of tests if their assertions are commented out?

Reply:
@vickywane 
We should have some sort of assertion test. 
Should this be a visit to the /timeline URL?
And what are we expecting to see once the page loads?

Reply:
That's a before hook event not really an assertion. It's what happens the first time the test case is run. In this case to visit the `timeline` page.